piano-hero/mobile/src/main/java/com/novoda/pianohero/GamePresenter.java

```diff
@@ -0,0 +1,44 @@
+package com.novoda.pianohero;
+
+class GamePresenter implements GameMvp.Presenter {
+
+    private final GameMvp.Model gameModel;
+    private final GameMvp.View view;
+
+    GamePresenter(GameMvp.Model gameModel, GameMvp.View view) {
+        this.gameModel = gameModel;
+        this.view = view;
+    }
+
+    @Override
+    public void onCreate() {
+        RoundViewModel viewModel = gameModel.startGame();
+        view.showRound(viewModel);
+    }
+
+    @Override
+    public void onNotesPlayed(Note... notes) {
+        RoundViewModel viewModel = gameModel.onNotesPlayed(new GameMvp.Model.Callback() {
```

Comment:
meh, too big a change for this PR - which is just refactoring, will ping you on next 1!